File: client/src/app/_dtos/hr/candidateAssessedShortDto.ts
export interface ICandidateAssessedShortDto
{
     candidateAssessmentId: number;
     orderItemId: number;
     customerName: string;
     requireInternalReview: boolean;
     candidateName: string;
     assessedByUsername: string;
     assessResult: string;
     assessedOn: Date;
     checklistHRId: number;
     categoryRef: string;
}

export class CandidateAssessedShortDto implements ICandidateAssessedShortDto
{
     candidateAssessmentId= 0;
     orderItemId= 0;
     customerName = '';
     requireInternalReview= false;
     candidateName= '';
     assessedByUsername= '';
     checklistHRId= 0;
     assessResult= '';
     assessedOn: Date = new Date();

     categoryRef = '';
}

File: api/DTOs/HR/CandidateAssessedShortDto.cs
using api.DTOs.Admin.Orders;

namespace api.DTOs.HR
{
    public class CandidateAssessedShortDto
    {
        public int CandidateAssessmentId { get; set; }
        public int OrderItemId { get; set; }
        public string CategoryRef { get; set; }
        public string CustomerName { get; set; }
        public int CandidateId {get; set;}
        public string CandidateName { get; set; }
        public DateTime AssessedOn { get; set; }
        public string AssessedByUsername {get; set;}
        public string RequireInternalReview {get; set;}
        public int ChecklistHRId {get; set;}
        public string AssessResult { get; set; } 
        public OrderItemBriefDto orderItemBriefDto {get; set;}
    }
}